content/beginner/080_scaling/deploy_hpa.md

```diff
@@ -24,7 +24,7 @@ helm install metrics-server \
 ```
 
 {{% notice warning %}}
-**Edit 2020-04-22**: some versions of EKS may get the following error once you [create the HPA in the next step](https://eksworkshop.com/scaling/test_hpa/)
+**Edit 2020-04-22**: some versions of EKS may get the following error once you [create the HPA in the next step](https://eksworkshop.com/beginner/080_scaling/test_hpa/)
```

Comment:
Any reason we need to manually post the edit history of the warning? It's technically already captured in the change history.

Reply:
Great. Fixed it in the follow-up commit.